### PR TITLE
Fix: Twitter logo in TweetCard component

### PR DESCRIPTION
**Pull request by monicakh**
## What kind of change does this PR introduce?

Bug fix

## What is the current behavior?

Twitter logo needs to be fixed as it was showing only a black icon:
image

## What is the new behavior?

image

### CODE REVIEW

**Review by fsansalvadore — DISMISSED** [review]

**Comment by fsansalvadore** [comment]
@monicakh Are you still not seeing a white X?

I noticed the X on your version now renders a bit off. I think it's the svg viewport.

File: apps/www/components/TweetCard.tsx [hunk 1]
```diff
@@ -46,12 +45,17 @@ function TweetCard(props: TweetCard) {
             style={{ background: '#000' }}
           >
             <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 32 32"
```
Comment (fsansalvadore) [inline apps/www/components/TweetCard.tsx:49]:
```suggestion
              viewBox="0 0 24 24"
```

**Review by fsansalvadore — CHANGES_REQUESTED** [review]
This should keep it the correct size

**Comment by monicakh** [comment]
@fsansalvadore Yeah it's still not rendering on my end but it's only happening on Chrome.

**Comment by Isaiah-Hamilton** [comment]
This issue seems to be working fine in production. Is this issue only happening on your computer, local dev or browser?

**Pull request closed without merging**